cmd/protoc-gen-go-grpc/main.go

```diff
@@ -32,13 +32,23 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"google.golang.org/grpc"
+	"os"
+	"path/filepath"
 
 	"google.golang.org/protobuf/compiler/protogen"
+
 )
 
 var requireUnimplemented *bool
 
 func main() {
+	if len(os.Args) == 2 && os.Args[1] == "--version" {
+		fmt.Fprintf(os.Stderr, "%v %v\n", filepath.Base(os.Args[0]), grpc.Version)
```

Comment:
`fmt.Fprintln(os.Stderr, filepath.Base(os.Args[0]), grpc.Version)`

That will automatically put a space between the two, and a newline at the end, and defaults to `%v` formatting.

Reply:
I suppose this code is copied from [protoc-gen-go](https://github.com/protocolbuffers/protobuf-go/blob/master/cmd/protoc-gen-go/main.go) that I linked in the original issue... so your comments would be equally relevant there I suppose.

Reply:
Thanks for pointing it out! Wondering do you mean there will be an extra(besides a space we already had) space between the two?

Reply:
Hmm. This will switch to single dash option (`-version`), which is different from the double dash `—version` in `protoc-gen-go`. That’s confusing.

Reply:
@meling , thanks for pointing it out! I was confused about the single dash and double dash in flag package.
I tested in my local, it supports both single and double dash options in this current code.
```
▶ ./protoc-gen-go-grpc -version  
protoc-gen-go-grpc 1.30.0

▶ ./protoc-gen-go-grpc --version 
protoc-gen-go-grpc 1.30.0
```

Reply:
@meling do you mean to only have double dash for `version` ?